### DISCUSSION TITLE
Misaligned Header on mobile browsers.

### DISCUSSION THREAD

**Discussion by code-reaper08** [npm/feedback#503] [Website] [answered]
## Problem:
Hello 👋 I found that the header at [npmjs.com](https://www.npmjs.com/) is slightly misaligned when viewed through mobile browsers. The word `community` gets cut. I've attached screenshots for reference.

## Screenshot:
![npm-website-bug](https://user-images.githubusercontent.com/64256342/127025024-464ff1f8-88e3-41cc-aa45-3c907e8c25c7.jpg)

I tested on both, Chrome and Firefox on Android, I'm not sure of Safari on iPhone.

### Actual CSS:
```
.c6c55db4 {
    display: block;
    padding: 10px;
    position: relative;
    font-family: 'Poppins';
    color: #000;
    letter-spacing: 0.015625em;
}
```
### expected CSS:

```
.c6c55db4 {
    display: block;
    padding: 10px 0px 10px 10px;
    position: relative;
    font-family: 'Poppins';
    color: #000;
    letter-spacing: 0.015625em;
}
```
This solution is totally my assumption of what causes this issue. I Just hope it gets resolved 😄

**Comment by MylesBorins** [comment, marked as answer]
hey @code-reaper08

I've opened a bug with the engineering team. Thanks for reporting

**Reply by code-reaper08** [reply]
Awesome @MylesBorins 😃

It's my pleasure
